index.html

```diff
@@ -3816,6 +3836,56 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
 </pre>
           </aside>
         </section>
+        <!-- Maintain a fragment named "mASt-chunk" to preserve incoming links to it -->
+
+        <section id="mASt-chunk">
+          <h2><span class="chunk">mASt</span> mastering metadata</h2>
+
+          <p>The four decimal values below correspond to the four-byte mASt chunk type field:</p>
+
+          <pre>
+109 65 83 116
+</pre>
+          <p>If present, the <span class="chunk">mASt</span> chunk specifies the mastering display's metadata which includes
+          primaries, white point, maximum luminance, and minimum luminance as specified in [[SMPTE ST 2086]].</p>
+
+          <p>The following specifies the syntax of the <span class="chunk">mASt</span> chunk:</p>
+
+          <table id="mASt-chunk-syntax" class="numbered simple">
+            <caption>
+              mASt chunk components
+            </caption>
+
+            <tr>
+              <th>Name</th>
+              <th>Size</th>
+            </tr>
+
+            <tr>
+              <td>Mastering display colour primaries</td>
+              <td>12 bytes</td>
+            </tr>
+
+            <tr>
+              <td>Mastering display white point chromaticity</td>
+              <td>4 bytes</td>
+            </tr>
+
+            <tr>
+              <td>Mastering display maximum luminance</td>
+              <td>4 bytes</td>
+            </tr>
+          </table>
+
+          <aside class="note">
+            The three colour primaries are 4 bytes each, totalling 12 bytes. They are ordered starting with the primary with the
+            largest x chromaticity, followed by the primary with the largest y chromaticity, followed by the remaining primary.
+            This corresponds to RGB order.
+          </aside>
+
+          <p>The <span class="chunk">mASt</span> chunk MUST come before the <a class="chunk" href="#11PLTE">PLTE</a> and <a class=
+          "chunk" href="#11IDAT">IDAT</a> chunks.</p>
```

Comment:
Originally, we had hoped the formatting tool would prevent newlines within tags.
It looks like HTML Tidy allows them. :'(